src/main/resources/default/assets/common/scripts/dialog.js.pasta

```diff
@@ -234,7 +231,7 @@
      * @type {Wizard} the wizard helper
      */
     dialog.Wizard = (function () {
-        function Wizard() {
+        function Wizard(_form) {
```

Comment:
Where is `_form` used? In the code below , it looks like `_form = document.createElement('form');` creates a new object anyways inside `Wizard.prototype.addStep`